mscore/shortcut.cpp

```diff
@@ -3246,11 +3246,11 @@ Shortcut Shortcut::_sc[] = {
       {
          MsWidget::SCORE_TAB,
          STATE_NORMAL | STATE_NOTE_ENTRY,
-         "add-parenthesis",
```

Comment:
Hmm, I didn't think of this to possibly apply to 2.1. Should I?

I think the differentiaton between brackets [] and parentheses {} is new in master